lib/release/notes/log.rb

```diff
@@ -81,11 +81,22 @@ def find_all_tags_and_log_all
           next unless previous_tag.present? &&
                       system_log(tag_from: previous_tag, tag_to: ta, label: all_labels).present?
 
-          digest_date date: date_humanized(date: System.tag_date(tag: ta))
+          date = date_humanized(date: System.tag_date(tag: ta)) # Does this need to be assigned to a var?
+          header_content(date, ta)
           copy_single_tag_of_activity(tag_from: previous_tag, tag_to: ta)
         end
       end
 
+      # @api private
+      def header_content(date, tag)
+        if valid_header_title? == "date"
```

Comment:
Typically a method name with `*?` would make us assume a boolean response.  While this works fine as is, I think we can optimize here. My suggestion would be to continue to return a string on this configuration method maybe call it `#header_title_type` and then use that in some way here.  Just a suggestion and feel free to differ:

```ruby
def header_content(**date_and_tag)
  digest_header(date_and_tag[header_title_type.to_sym])
end
```